src/__vendor/launchpad.js

```diff
@@ -0,0 +1,23 @@
+/**
+ *  Loads the Launchpad js tracker (used by Redplanet) into the page,
+ *  and creates the global "launchpad" method
+ */
+export const launchpad = () => {
+	;(function (p, l, o, w, i, n, g) {
```

Comment:
Don't think the `;` is doing anything useful here:

```suggestion
	(function (p, l, o, w, i, n, g) {
```